cmd/checkMigrations.go

```diff
@@ -0,0 +1,230 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	repo   string
+	branch string
+)
+
+// NamedFile interface represents a local file object, with a Name function
+type NamedFile interface {
+	Name() string
+}
+
+type Set = map[string]bool
+
+type githubJSON struct {
+	Name string
+}
+
+type GithubFile struct {
+	githubJSON
+}
+
+func (file *GithubFile) UnmarshalJSON(b []byte) error {
+	return json.Unmarshal(b, &file.githubJSON)
+}
+
+func (file GithubFile) Name() string {
+	return file.githubJSON.Name
+}
+
+var checkMigrationsCmd = &cobra.Command{
+	Use:    "checkMigrations",
+	PreRun: initGithubParams,
+	Short:  "Check that the migrations in this repository are properly timestamped for a merge.",
+	Long: `Check that any new migrations in this branch will run after all the migrations in the
+staging branch`,
+	Run: func(cmd *cobra.Command, args []string) {
+		err := checkMigrations()
+
+		if err != nil {
+			fmt.Printf("Error checking migrations %s\n", err.Error())
+			os.Exit(1)
+		}
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(checkMigrationsCmd)
+	checkMigrationsCmd.Flags().String("repo", "makerdao/vdb-mcd-transformers", "Github Repository to check against, must be public")
+	checkMigrationsCmd.Flags().String("branch", "staging", "Branch to check against")
+
+	viper.BindPFlag("repo", checkMigrationsCmd.Flags().Lookup("repo"))
+	viper.BindPFlag("branch", checkMigrationsCmd.Flags().Lookup("branch"))
+}
+
+func initGithubParams(cmd *cobra.Command, args []string) {
+	repo = viper.GetString("repo")
+	branch = viper.GetString("branch")
+}
+
+func checkMigrations() error {
+	stagingMigrations, err := getGithubFileNames()
+
+	if err != nil {
+		return err
+	}
+
+	localMigrations, err := getLocalMigrations()
+
+	if err != nil {
+		return err
+	}
+
+	newMigrations := NewMigrations(localMigrations, stagingMigrations)
+	logrus.Println("New Migrations are", newMigrations)
```

Comment:
does this look alright when it prints? Would have expected to see a `: ` after "are"